<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Abilities</class>
 <widget class="QWidget" name="Abilities">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>581</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabStr">
    <attribute name="title">
     <string>Strength</string>
    </attribute>
    <widget class="QSpinBox" name="spinStr">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>57</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblStrLevel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Strength Level:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textStr">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>531</width>
       <height>191</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot;&gt;This stat reflects your character’s ability to perform tasks that require brute physical activity.  It directly impacts the physical damage that your character deals.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblStrAbout">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>About Strength Ability</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabDexerity">
    <attribute name="title">
     <string>Dexerity</string>
    </attribute>
    <widget class="QLabel" name="lblDexAbout">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>About Dexerity Ability</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textDex">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>531</width>
       <height>191</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot;&gt;This stat reflects your character’s ability to perform tasks that require quick, deft, or accurate movements.  It directly impacts your ability to dodge.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblDexLevel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dexerity Level:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinDex">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>57</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabConstruction">
    <attribute name="title">
     <string>Constitution</string>
    </attribute>
    <widget class="QSpinBox" name="spinCon">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>57</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblConLevel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Constitution Level:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textCon">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>531</width>
       <height>191</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot;&gt;This stat reflects your character’s fortitude; your character’s ability to receive damage, both physical and magical.  It directly impacts your ability to perform tasks that require extended stamina.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblConAbout">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>About Constitution Ability</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabIntelligence">
    <attribute name="title">
     <string>Intelligence</string>
    </attribute>
    <widget class="QSpinBox" name="spinInt">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>57</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblIntLevel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Intelligence Level:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textInt">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>531</width>
       <height>191</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot;&gt;This stat represents your character’s capacity for learning and your character’s amassed knowledge. It directly impacts your ability to perform mental feats and cast spells effectively.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblIntAbout">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>About Intelligence Ability</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWizdom">
    <attribute name="title">
     <string>Wisdom</string>
    </attribute>
    <widget class="QSpinBox" name="spinWiz">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>57</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblWizLevel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Wisdom Level:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textWiz">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>531</width>
       <height>191</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot;&gt;This stat reflects your character’s natural intuition.  It directly impacts your Will saving throws, as well as your ability to perform street-wise skills and spirituality-based skills.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblWizAbout">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>About Wisdom Ability</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabCharism">
    <attribute name="title">
     <string>Charisma</string>
    </attribute>
    <widget class="QSpinBox" name="spinCha">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>57</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblChaLevel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Charisma Level:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textCha">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>531</width>
       <height>191</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot;&gt;This stat reflects your character’s ability to communicate.  It directly impacts the skills involving façade and persuasion.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblChaAbout">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>About Charisma Ability</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>470</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAboutClasses">
   <property name="geometry">
    <rect>
     <x>483</x>
     <y>10</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>About Abilities</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>470</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>34</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Abilities</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRoll">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Roll</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
